junit-jupiter-params/src/test/kotlin/ParameterizedTestNameFormatterIntegrationTests.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.TestInfo
+import org.junit.jupiter.api.TestMethodOrder
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+
+@TestMethodOrder(MethodOrderer.MethodName::class)
+class ParameterizedTestNameFormatterIntegrationTests {
+
+    @MethodSource("methodSource")
```

Comment:
Why did you choose the more complicated setup for `@MethodSource` with a companion object and `@JvmStatic`?

I'd likely go with `@ValueSource(strings = ["foo", "bar"])` (assuming that's the correct syntax for Kotlin).

Reply:
The example that the author showed used that setup. But you're right I'll change it to a simpler setup.